lib/src/RTCSession.dart

```diff
@@ -2266,8 +2266,8 @@ class RTCSession extends EventEmitter {
 
       // Be ready for 200 with SDP after a 180/183 with SDP.
       // We created a SDP 'answer' for it, so check the current signaling state.
-      //if (this._connection.signalingState == RTCSignalingState.RTCSignalingStateStable)
-      /*{
+      if (this._connection.signalingState ==
+          RTCSignalingState.RTCSignalingStateStable) {
```

Comment:
changing this line to 

> if (this._connection.signalingState ==
>           RTCSignalingState.RTCSignalingStateHaveLocalOffer)

seems to work.

Note this change also requires a change in flutter-webrtc to make **this._connection.signalingState** accessable.